mlir/lib/IR/MLIRContext.cpp

```diff
@@ -1044,39 +1044,6 @@ AbstractType::lookup(StringRef name, MLIRContext *context) {
 /// This should not be used directly.
 StorageUniquer &MLIRContext::getTypeUniquer() { return getImpl().typeUniquer; }
 
-Float4E2M1FNType Float4E2M1FNType::get(MLIRContext *context) {
```

Comment:
We also need to remove the fields from the ContextImpl (and their initialization)